cannon/mipsevm/memory/memory.go

```diff
@@ -245,7 +246,7 @@ func (m *Memory) GetMemory(addr Word) uint32 {
 }
 
 // GetWord reads the maximum sized value, [arch.Word], located at the specified address.
-// Note: Also known by the MIPS64 specification as a "double-word" memory access.
+// Note: Also referred to by the MIPS64 specification as a "double-word" memory access.
```

Comment:
(nit) this is only true for 64-bit right?:
```suggestion
// Note: In 64-bit architectures, corresponds to the MIPS64 specification's "doubleword" memory access.
```